### PR TITLE
Merge future to master

### PR DESCRIPTION
**Pull request by nguerrera**
We're opening master for 15.3 / netstandard 2.0 development, so merging future there and we will delete the future branch.

@srivatsn @ramarag @eerhardt @333fred

### CODE REVIEW

**Review by natidea — APPROVED** [review]

**Comment by nguerrera** [comment]
I'll wait for @srivatsn's approval just to be absolutely sure we're ready to pull the trigger on future -> master.

**Review by livarcocc — APPROVED** [review]

**Review by srivatsn — APPROVED** [review]

**Pull request merged**
